notebooks/Solutions/Causal Interpretation for Ames Housing Price.ipynb

```diff
@@ -2041,9 +2049,16 @@
     "### Direct Causal Effect --  Do certain house characteristics have a direct effect on home value?"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We've seen which features are good predictors of home value, but are these features also causally signficiant? Below we use the CausalAnalysis class to assess the causal effects of certain house characteristics on home value."
```

Comment:
This is good, but it would be nice to have a parallel statement after calling CausalAnalysis which summarizes whether it agrees or disagrees with the predictive analysis.